### PR TITLE
refactor(rxPromiseNotifications): FRMW-402-convert to quark service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-402

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxPromiseNotifications` to Quark service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `rxPromiseNotifications` Quark Service

### Demo Page
screen shot 2015-12-28 at 2 29 09 pm

### Demo Video 
rxpromisenotifications-demo

### API Docs
screen shot 2015-12-28 at 2 28 14 pm

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <div layout="row">
+
+    <div rx-form flex>
+      <rx-form-section>
+        <div>
+          <h2>Example</h2>
+          <p>Select <code>Create Service</code> to simulate creating a service, then select
+             <code>Resolve Promise</code> or <code>Reject Promise</code> to simulate success or failure.
+          </p>
+          <button class="button" ng-click="addPromise()">Create service</button>
```
Comment (CITguy) [inline src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.html]: I'd change wording to `Create Promise` to reflect actual functionality.

File: src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+angular.module('demoApp')
+.controller('rxPromiseNotificationsSimpleCtrl', function ($rootScope, $scope, $window, rxNotify, 
+                                                    rxPromiseNotifications, $q) {
```
Comment (CITguy) [inline src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.js]: I'm not sure if the following are needed as arguments.

* `$rootScope` - unused
* `$window` - unused

**Comment by 100stacks** [comment]
@CITguy comments implemented.  I also cleaned up the wording in the API Docs in https://github.com/rackerlabs/encore-ui/pull/1476#issuecomment-167662407 :+1:

File: src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<div ng-controller="rxPromiseNotificationsSimpleCtrl">
+  <div layout="row">
+
```
Comment (CITguy) [inline src/quarks/rxPromiseNotifications/docs/examples/rxPromiseNotifications.simple.html]: please remove blank line

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Comment by 100stacks** [comment]
@parlarjb rebased :+1:

**Pull request merged**